tests/misc.rs

```diff
@@ -103,25 +103,6 @@ use thiserror::Error;
 // Cargo.toml file (see comments above), so here we use just the hardcoded bytecode instructions
 // instead.
 
-#[cfg(not(windows))]
```

Comment:
Invalid test?

Reply:
test_vm_ldabsb was the one duplicated and now merged back into test_vm_jit_ldabsb